### PR TITLE
Fix OWASP by bumping various library versions

### PR DESCRIPTION
**Pull request by symposion**
Bump some key libraries. I've had to do a bit of a hack for GSON (exclude from the upper bound dep checker) until Gabor can release a new Approvalcrest. I think the fix is trivial so hopefully he can do that soon.

### CODE REVIEW

**Review by idasbiste — DISMISSED** [review]

**Comment by idasbiste** [comment]
Meh, there are some upper bound dependencies issues.

**Review by mrpeny — DISMISSED** [review]

**Review by pkbdevdeploy — COMMENTED** [review]
Version `60-head-5f72158-266972-SNAPSHOT` was generated from commit `5f721583213c21686db528bc1bcf1eed6dec25f4`

File: pom.xml [hunk 1]
```diff
@@ -407,7 +446,11 @@
                                     to pick the highest version. Otherwise we may end up unexpectedly swapping to an older
                                     version if the 'nearest' definition changes, or end up unexpectedly using a different version
                                     to what we've specified via a parent bom -->
-                                    <requireUpperBoundDeps/>
+                                    <requireUpperBoundDeps>
+                                        <excludes>
+                                            <exclude>com.google.code.gson:gson</exclude>
+                                        </excludes>
+                                    </requireUpperBoundDeps>
```
Comment (idasbiste) [inline pom.xml:453]: 👍

**Review by idasbiste — APPROVED** [review]

**Review by mrpeny — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (mrpeny) [reply pom.xml:453]: I'll resolve this if you don't mind 😃

**Comment by symposion** [comment]
EEEEK. Never merge one of these PRs until the PHR pr is  green

**Comment by mrpeny** [comment]
Sorry for the inconvenience! I have checked if any `dont-merge-yet` labels were present and proceeded with the merge but I will keep it in mind for the future.

**Review by pkbdevdeploy — COMMENTED** [review]
Version `60-ad4923e-267029` was generated from commit `ad4923ef167e535dcd69e07868c72451c7ee56da`

**Pull request merged**